pretix_servicefees/views.py

```diff
@@ -9,7 +9,9 @@
 
 class ServiceFeeSettingsForm(SettingsForm):
     service_fee_abs = forms.DecimalField(label=_('Service fee'))
+    service_fee_percent = forms.DecimalField(label=_('Service fee (%)'))
```

Comment:
The ``total`` argument of the fees signal that we use above only takes into account the sum of all cart positions, not the contributions of any other fees, since the calculation order of different fees is not defined. We should at least specify this in a ``help_text`` attribute, such as:

```
Note that this percentage will currently only be calculated on the summed price of sold tickets, not on other fees like e.g. shipping fees, if there are any.
```